docs/api/geo_datasets.csv

```diff
@@ -1,23 +1,23 @@
-Dataset,Type,Source,Size (px),Resolution (m)
-`Aboveground Woody Biomass`_,Masks,"Landsat, LiDAR","40,000x40,000",30
-`Aster Global DEM`_,Masks,Aster,"3,601x3,601",30
-`Canadian Building Footprints`_,Geometries,Bing Imagery,-,-
-`Chesapeake Land Cover`_,"Imagery, Masks",NAIP,-,1
-`Global Mangrove Distribution`_,Masks,"Remote Sensing, In Situ Measurements",-,3
-`Cropland Data Layer`_,Masks,Landsat,-,30
-`EDDMapS`_,Points,Citizen Scientists,-,-
-`EnviroAtlas`_,"Imagery, Masks","NAIP, NLCD, OpenStreetMap",-,1
-`Esri2020`_,Masks,Sentinel-2,-,10
-`EU-DEM`_,Masks,"Aster, SRTM, Russian Topomaps",-,25
-`GBIF`_,Points,Citizen Scientists,-,-
-`GlobBiomass`_,Masks,Landsat,"45,000x45,000",100
-`iNaturalist`_,Points,Citizen Scientists,-,-
-`L7 Irish`_,"Imagery, Masks",Landsat,"8,400x7,500","15, 30"
-`L8 Biome`_,"Imagery, Masks",Landsat,"8,900x8,900","15, 30"
-`LandCover.ai Geo`_,"Imagery, Masks",Aerial,"4,200--9,500",0.25--0.5
-`Landsat`_,Imagery,Landsat,"8,900x8,900",30
-`NAIP`_,Imagery,Aerial,"6,100x7,600",1
-`NLCD`_,Masks,Landsat,-,30
-`NCCM`_,Masks,Sentinel-2,-,10 
-`Open Buildings`_,Geometries,"Maxar, CNES/Airbus",-,-
-`Sentinel`_,Imagery,Sentinel,"10,000x10,000",10
+Dataset,Type,Source,License,Size (px),Resolution (m)
+`Aboveground Woody Biomass`_,Masks,"Landsat, LiDAR","CC-BY-4.0","40,000x40,000",30
+`Aster Global DEM`_,Masks,Aster,"Assumed CC0-1.0","3,601x3,601",30
+`Canadian Building Footprints`_,Geometries,Bing Imagery,"ODbL-1.0",-,-
+`Chesapeake Land Cover`_,"Imagery, Masks",NAIP,"Assumed CC0-1.0",-,1
+`Global Mangrove Distribution`_,Masks,"Remote Sensing, In Situ Measurements","Assumed CC0-1.0",-,3
+`Cropland Data Layer`_,Masks,Landsat,"CC0-1.0",-,30
+`EDDMapS`_,Points,Citizen Scientists,"Assumed CC0-1.0",-,-
+`EnviroAtlas`_,"Imagery, Masks","NAIP, NLCD, OpenStreetMap","CC-BY-4.0",-,1
+`Esri2020`_,Masks,Sentinel-2,"CC-BY-4.0",-,10
+`EU-DEM`_,Masks,"Aster, SRTM, Russian Topomaps",-,-,25
```

Comment:
They have their own [custom license](https://spacedata.copernicus.eu/documents/20123/121286/CSCDA_ESA_Mission-specific+Annex_31_Oct_22.pdf/fb109818-56ad-bbee-053c-d972aed25ce6?t=1674741175657), maybe call it CSCDA-ESA.